src/provider.ts

```diff
@@ -21,6 +21,7 @@ const BuiltInProviders = [
   'ipx',
   'netlify',
   'layer0',
```

Comment:
we removed provider...

Reply:
Wouldn't that be breaking one?

Reply:
Image v1 is not officially released yet but of course we can also support it as alias if you like...